chaosaws/ecs/actions.py

```diff
@@ -155,3 +213,31 @@ def list_tasks(cluster: str, service: str, client: boto3.client) -> List[str]:
                                 nextToken=next_token, maxResults=100)
         tasks.extend(res["taskArns"])
     return tasks
+
+def list_running_tasks_in_cluster(cluster: str, client: boto3.client, service: str = None):
+    if service:
+        res = client.list_tasks(cluster=cluster, serviceName=service,
```

Comment:
I would assume the conditional isn't needed here.

If `service` is unset, then it will be passed as `None` (well as falsey mind you) to `serviceName` as if it had not been set at all.

Reply:
Passing serviceName as None actually errors out unfortunately :(

Reply:
Gotcha. Odd choice from the underlying API but there we go :)